cfy_manager/components/usage_collector/scripts/script_utils.py

```diff
@@ -42,26 +44,38 @@ def get_storage_manager_instance():
         config.reset(config.Config())
 
 
-def create_manager_id_file():
-    if path.exists(MANAGER_ID_PATH):
-        with open(MANAGER_ID_PATH) as f:
-            existing_manager_id = f.read().strip()
-            if existing_manager_id:
-                return
-    with open(MANAGER_ID_PATH, 'w') as f:
-        f.write(uuid4().hex)
+def needs_to_send_data(interval_type):
+    timestamp, interval_sec = _get_timestamp_and_interval(interval_type)
+    time_to_update = (timestamp + interval_sec + BUFFER_TIME) < time.time()
+    if (timestamp is None) or time_to_update:
+        return True
+    return False
+
+
+def _get_timestamp_and_interval(interval_type):
+    with open(USAGE_CONFIG_PATH) as usage_config_file:
+        usage_config = json.load(usage_config_file)
+    interval = usage_config.get(interval_type)
+    if interval_type == HOURS_INTERVAL:
+        timestamp = usage_config.get(HOURLY_TIMESTAMP)
+        interval_sec = interval*60*60
```

Comment:
huh, this passes flake? as opposed to `60 * 60` with spaces?

Reply:
Yep, but I'll change it for readability, thanks.